.github/workflows/release.yaml

```
@@ -0,0 +1,91 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      target-branch:
+        description: "Branch to release"
+        required: true
+        type: choice
+        default: 'main'
+        options:
+        - main
+      version:
+        description: "Version to release (e.g. 1.1.0 or 1.2.0.alpha1)"
```

```suggestion
        description: "Version to release (e.g. 1.1.0 or 1.2.0-alpha1)"
```
Must be adapted when changed

---

done